app/components/MediaContainer.tsx

```diff
@@ -0,0 +1,25 @@
+'use client'
+
+import React from 'react'
+import { useSyncedStore } from '@syncedstore/react'
+
+import MediaPlayer from '@/app/components/MediaPlayer'
+import { store } from '@/app/store'
+
+const MediaContainer = () => {
+  const state = useSyncedStore(store)
+
+  return (
+    <div>
+      {state.player_state?.playing_youtube_id ? (
+        // added the key so it re-renders the MediaPlayer component
+        // when same url is played
+        <MediaPlayer url={state.player_state.playing_youtube_id} initTime={0} />
```

Comment:
did you intend to get rid of the key prop in this component as I had it setup in `page.tsx` previously? I'm getting that issue where if we have the same video twice, it doesn't reload new state as if we are opening a brand new url

Reply:
Good catch! I had removed it earlier on in my changes because I'd assumed I would fix it by adding the start time functionality, but then I made https://github.com/jonvuri/tonlist/issues/10.

I just added in a super basic form of that that doesn't really do anything except reimplement a key to fix this issue again. It sets the player start time to the current time every time we hit 'Play next'.